tealer/teal/instructions/instructions.py

```diff
@@ -513,8 +517,17 @@ def __str__(self) -> str:
 class Callsub(InstructionWithLabel):
     def __init__(self, label: str):
         super().__init__(label)
+        self._return_point: Optional[Instruction] = None
         self._version: int = 4
 
+    @property
+    def return_point(self) -> Optional[Instruction]:
+        return self._return_point
+
+    @return_point.setter
+    def return_point(self, ins: Instruction) -> None:
```

Comment:
Can you change the return point more than once?
Otherwise you could:
```
if self._return_point is not None:
    raise ValueError("Return_point already set") 
```

Reply:
return_point is only used while finding the retsubs corresponding to a subroutine other then that they aren't used anywhere else. they will be set only once, adding that check makes sense.